test/e2e/snaps/test-snap-bip-32.spec.ts

```diff
@@ -119,18 +117,16 @@ describe('Test Snap bip-32', function () {
         );
 
         // Select an invalid (non-existent) entropy source, enter a message, sign, approve and validate the result
-        await testSnaps.scrollAndSelectEntropySource(
-          'bip32EntropyDropDown',
-          'Invalid',
-        );
+        await testSnaps.selectEntropySource('bip32EntropyDropDown', 'Invalid');
         await testSnaps.fillMessage('messageSecp256k1Input', 'bar baz');
         await testSnaps.clickButton('signBip32messageSecp256k1Button');
 
-        // Check the error message and close the alert.
-        await driver.waitForAlert(
-          'Entropy source with ID "invalid" not found.',
-        );
-        await driver.closeAlertPopup();
```

Comment:
intentionally -> don't close the alert. This can lead again to race conditions in the case we couldn't switch windows before the alert appeared. Then there's going to be no way to close the alert.
Given that we already validated the alert text, we don't care on closing it or leaving it open, as this is not relevant for the wallet